--- theme_original.inc	2017-08-11 19:04:46.876822375 -0300
+++ theme_atech.inc	2017-08-11 19:04:55.596866822 -0300
@@ -2215,13 +2215,21 @@
         // Render nested list.
         $data .= theme_item_list(array('items' => $children, 'title' => NULL, 'type' => $type, 'attributes' => $attributes));
       }
-      if ($i == 1) {
-        $attributes['class'][] = 'first';
+      $mainDivAttributes['class'][] = "panel";
+      $mainDivAttributes['class'][] = "panel-default";
+      $attributes['class'][] = "panel-heading";
+      switch ($i) {
+        case 1:
+          $attributes['class'][] = 'first';
+          break;
+        
+        case $num_items:
+          $attributes['class'][] = 'last';
+          break;
       }
-      if ($i == $num_items) {
-        $attributes['class'][] = 'last';
-      }
-      $output .= '<li' . drupal_attributes($attributes) . '>' . $data . "</li>\n";
+      $output .= '<div' . drupal_attributes($mainDivAttributes) . '>';
+      $output .= '<div' . drupal_attributes($attributes) . '>' . $data . '</div>';
+      $output .= '</div>';
     }
     $output .= "</$type>";
   }
